cluster.tf

```diff
@@ -397,6 +397,11 @@ resource "google_container_cluster" "primary" {
   }
 
   lifecycle {
+    precondition {
+      condition     = var.ip_range_services == null && var.kubernetes_version != "latest" ? tonumber(split(".", var.kubernetes_version)[0]) >= 1 && tonumber(split(".", var.kubernetes_version)[1]) >= 29 : true
```

Comment:
Can we verify this condition (using variables) is supported in Terraform v1.3 (currently the minimum required version)